tools/run_satwnds/README.md

```diff
@@ -0,0 +1,23 @@
+# Run Sat Winds
+
+Split a sat wind bufr file into its subsets, and runs bufr2ioda.x on each one 
+with the proper configuration for that subset.
+
+## Dependencies
+
+* **Python 3.6+**
+
+The following executables must be available in the shell path.
+
+* **bufr2ioda.x** - From ioda_converters _feature/bufr-enhanced-exports_
+* **split_by_subset.x** - From NCEPLibs-Bufr _feature/split_by_subset_
```

Comment:
feature/split_by_subset has been merged to NOAA-EMC/NCEPLIBS-bufr/develop
feature/bufr-enhanced-exports will be merged soon.